src/js/auth/containers/LoginApp.jsx

```diff
@@ -24,6 +24,8 @@ class LoginApp extends React.Component {
     }).then(response => {
       return response.json();
     }).then(json => {
+      // console.log(json);
+      window.opener.localStorage.clear();
```

Comment:
what if something else is in localStorage, does this remove it? (and is that desired?)